features/mixed-content.yml

```
@@ -1,7 +1,9 @@
-draft_date: 2024-09-09
-name: Mixed Content
-description: TODO
+´name: Mixed content handling
+description: If a document is loaded over HTTPS and any of its resources are loaded over an insecure protocol, then depending on the resource type, the browser will either attempt to load the insecure resource over HTTPS or block it.
```

There are lots of conditions on this sentence. Maybe something like this would be easier to follow? I'm not sure I succeeded.

```suggestion
description: Mixed content is a document that is loaded over HTTPS but uses resources over an insecure protocol. Browsers attempt to load mixed content over HTTPS, but otherwise block insecure resources, especially sensitive ones such as scripts and style sheets.
```

---

Thank you, @wbamberg. That's much better. Adjusted slightly for the style guide here:

```suggestion
description: When a document is loaded over HTTPS, browsers ensure that none of the document's resources are loaded over an insecure protocol. Instead, resources that the document attempts to load over an insecure protocol are either loaded over HTTPS or are blocked.
```